packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -550,8 +550,8 @@ const finish = async ({ state }) => {
  * @param {import('@agoric/governance/src/contractGovernance/typedParamManager').TypedParamManager<import('./params.js').VaultDirectorParams>} directorParamManager
  * @param {ZCFMint<"nat">} debtMint
  */
-const makeVaultDirector = defineKindMulti(
-  'VaultDirector',
+const makeVaultDirector = defineDurableKindMulti(
+  makeKindHandle('VaultDirector'),
```

Comment:
This would make a new handle if the vaultDirector were ever revived. 

The up-to-date approach is there's a `baggage` parameter, and you use `provide` to retrieve the handle if it already exists.

```suggestion
const vaultDirectorKindHandle = provideKindHandle(baggage, 'vaultDirector');
const makeVaultDirector = defineDurableKindMulti(
  vaultDirectorKindHandle,
```

Reply:
> This would make a new handle if the vaultDirector were ever revived.

That will only happen for reasons other than upgrade? The scope of this PR durability but not upgradability in place. (i.e. that the necessary data is available in storage)

When `provideKindHandle` is in master, I could use it with empty baggage and have a TODO to provide baggage when making it upgradeable. How's that sound?

Reply:
I'm not sure where the cut-off for durability vs. upgradeability is. I can see an argument for stopping with what you have, but I can also argue that if the goal is upgradeability that's plausible to a reviewer, though not tested, I think we should get as far as wiring up the code to reinitialize the handleKinds.

@erights, do you have an opinion?

Reply:
Recapping from other discussions: MN-1 goal is "reachably durable" and "durable" is a milestone towards that. The scope of this PR is just "durable" (no baggage).